libs/datamodel/core/tests/reformat/reformat_implicit_relations.rs

```diff
@@ -41,6 +41,33 @@ model Post2 {
     assert_reformat(input, expected);
 }
 
+#[test]
+fn back_relation_fields_and_directive_must_be_added_even_when_directive_is_missing() {
```

Comment:
Just reread this test file. There's one ignored test case in it. I think this one should work now.

Reply:
And the test name is wrong i think. Should be something like `must_add_relation_directive_to_existing_relation_fields`.

Reply:
Just gave it a try - does not seem to work yet.

Reply:
The new logic only checks if the directive exists, not if every single field inside the relation is there yet.